test/mocks.cpp

```diff
@@ -103,6 +105,11 @@ int mock_pnal_eth_send (pnal_eth_handle_t * handle, pnal_buf_t * p_buf)
    return p_buf->len;
 }
 
+int mock_pnal_eth_recv (void * arg, pnal_buf_t * buf)
```

Comment:
This seems backwards? The purpose of a mock-function is to decouple from external dependencies. This instead adds an external dependency?

Reply:
I have updated the mock implementation.
Done